app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title
+    base_title = t "layouts.application.title"
+    if page_title.empty?
+      base_title
+    else
+      page_title + " | " + base_title
```

Comment:
```suggestion
      [page_title, base_title].join(" | ")
```
hạn chế + string e nhé